src/content/lesson/everything-you-need-to-start-using-sqlalchemy.md

```diff
@@ -115,3 +115,21 @@ person = Person.query.get(3)
 person.name = "Bob"
 db.session.commit()
 ```
+## Transactions
+
+A transaction is a sequence of operations(like INSERT, UPDATE, SELECT) made on your database performed as one single work of unit. In other words a transaction will never be complete unless each individual operation within the group is succesful. If any operation within the transaction fails, the entire transaction fails, the entire transaction fails.
+
+Transactions have the following 4 standard properties(known as ACID properties):
+
+![Transactions](../../assets/images/tran-1.png)
+
+A transaction ends with COMMIT or ROLLBACK. 
+
+On a datababase the COMMIT command should be issued so that changes will take effect (COMMIT works just like on github, you add every file and modification first  `git add` and then by saying `git commit` you save your changes.) 
```

Comment:
```suggestion
On a datababase the COMMIT command should be issued, so that changes will take effect. COMMIT works just like on Github (you add every file and modification first  `git add` and then by saying `git commit` you save your changes).
```